crates/viewer/re_viewer_context/src/command_sender.rs

```diff
@@ -8,6 +8,7 @@ use re_ui::{UICommand, UICommandSender};
 
 /// Commands used by internal system components
 // TODO(jleibs): Is there a better crate for this?
+#[derive(strum_macros::Display)]
```

Comment:
No, some variants have eminently non-`Debug` content.

Reply:
but.. why would those in turn have Display rather than Debug? Obviously you want to debug them, so we should have Debug trickle down rather than Display?

Reply:
I did that and it does seem cleaner.